specs/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@eth-optimism/specs",
+  "version": "0.1.0",
+  "license": "MIT",
+  "private": true,
+  "devDependencies": {
+    "prettier": "2.2.1"
+  },
+  "scripts": {
+    "format": "yarn prettier --write \"{l2geth,protocol}/**/*.md\"",
+    "format:check": "yarn prettier --check \"{l2geth,protocol}/**/*.md\""
```

Comment:
```suggestion
    "lint:fix": "yarn prettier --write \"{l2geth,protocol}/**/*.md\"",
    "lint": "yarn prettier --check \"{l2geth,protocol}/**/*.md\""
```

Reply:
Nice, I'd considered exactly this change. :)